dev/com.ibm.ws.componenttest/src/componenttest/app/FATServlet.java

```diff
@@ -57,12 +57,21 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             try {
                 before();
 
+                // Use reflection to try invoking various test method signatures:
+                // 1)  method(HttpServletRequest request, HttpServletResponse response)
+                // 2)  method()
+                // 3)  method(PrintWriter output)
```

Comment:
I don't see 3 or 4 in this list being tried

Reply:
oops, had different method signatures initially and then I decided to go with the `invokeTest()` approach instead of guessing various test method signatures.  Will clean up this comment